.travis.yml

```diff
@@ -1,21 +1,69 @@
 sudo: required
+dist: trusty
 language: php
-php:
```

Comment:
I know I'm totally splitting hairs here, but can we try to use both the `php` and `matrix` blocks a la https://github.com/Islandora-Labs/islandora_solution_pack_oralhistories/blob/7.x/.travis.yml ?